Gemfile

```diff
@@ -13,8 +13,9 @@ group :development, :test do
   gem 'climate_control'
 end
 
-gem 'azure_mgmt_resources', '~>0.5.0'
-gem 'azure_mgmt_compute',   '~>0.5.0'
-gem 'azure_mgmt_network',   '~>0.5.0'
-gem 'azure_mgmt_storage',   '~>0.5.0'
-gem 'haikunator',           '~>1.1'
\ No newline at end of file
+gem 'azure_mgmt_resources', '~>0.6.0'
+gem 'azure_mgmt_compute',   '~>0.6.0'
+gem 'azure_mgmt_network',   '~>0.6.0'
+gem 'azure_mgmt_storage',   '~>0.6.0'
+gem 'haikunator',           '~>1.1'
+
```

Comment:
Here is the interesting detailes: I used RuboCop to find out what should be the expected behaviors for the line endings. Here is the `.rubocop` configuration [Reference: [.rubocop.yml](https://github.com/rails/rails/blob/master/.rubocop.yml#L27)]

```
# No extra empty lines.
Style/EmptyLines:
  Enabled: true
```

**Current condition on the master - Gemfile**
![master_branch](https://cloud.githubusercontent.com/assets/8782776/18888079/b02fc470-84ab-11e6-93c6-dd09bce34254.PNG)

**Condition with your PR branch - Gemfile**
![sarangan12](https://cloud.githubusercontent.com/assets/8782776/18888089/b95bb036-84ab-11e6-814f-c08973386df3.PNG)

**Condition with your PR branch removing extra new line - Gemfile**
![deleteline](https://cloud.githubusercontent.com/assets/8782776/18888096/c2930b22-84ab-11e6-9d63-a2e57d42d52f.PNG)

Rubocop sees extra new line (2 CR LF) as offense on having an extra line but not when 1 CR LF feed at the end of file. It's really interesting. What do you think?